README.md

```diff
@@ -39,6 +39,14 @@ $ brew install faas-cli
 
 Note: The `brew` release may not run the latest minor release but is updated regularly.
 
+Via npm:
+
+```
+$ npm install --global @openfaas/faas-cli
```

Comment:
what about `-g` that is more common?

Who owns `@openfaas`, is that the account I have? How are we pushing to it and auth'ing to npm during CI for this?

Reply:
`@openfaas` is a [user](https://www.npmjs.com/~openfaas) on NPM. I thought it was an org already. There are ways to auth an push during CI, but I'm not positive on how that's done.

For the `-g`, I usually spell out the entire word in documentation so there isn't any confusion over what it stands for. I'm open to making it just `-g` though.

Reply:
This would be a use for adding a `.npmrc` file to the repo (in the same directory as the `package.json` file)
There, you would add the auth token that you would need to generate locally first. Of course, you would want to keep that in a (TravisCI secret](https://docs.travis-ci.com/user/deployment-v2#sts=Securing%20secrets%20#)

My thoughts:
* Run `npm login` locally for the `@openfaas` user
* Get the auth token generated: `cat ~/.npmrc`
* Use Travis CLI to encrypt that token: `travis encrypt <auth token>`
* Add that encrypted value to the travis build's environment
* Add the `.npmrc` file and reference that env var 
Examples here: https://blog.npmjs.org/post/118393368555/deploying-with-npm-private-modules